src/transaction/log_page_buffer.c

```diff
@@ -1913,7 +1913,11 @@ logpb_copy_page (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_CS_ACCESS_MODE
   else
     {
       er_set (ER_FATAL_ERROR_SEVERITY, ARG_FILE_LINE, ER_LOG_PAGE_CORRUPTED, 1, pageid);
-      return ER_LOG_PAGE_CORRUPTED;
+      if (log_csect_entered)
+        {
+          LOG_CS_EXIT (thread_p);
+        }
+	  return ER_LOG_PAGE_CORRUPTED;
```

Comment:
how about handling an error like other code? 
`rv = ER_LOG_PAGE_CORRUPTED;` 
`goto exit;`

Reply:
I agree!

Reply:
And use the indent tool to match the code-style.

Reply:
@joohok @wrlawodms @hornetmj 
Thank you for your comments.
I have changed my commit.